### DISCUSSION TITLE
Import App to Budibase from Github? (Or some kind of version control)

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#5194] [Help] [answered]
Is it possible to manually or automate an import of a budibase app from a Github repo? I can't find anything in the docs about a process like this that could help with Version Control.

I guess more importantly, is there a way to handle version control other than manual updates and publishing.

**Comment by melohagan** [comment, marked as answer]
Hi @CosmasSC128 

At the moment there is not version control for budibase, however it is on the roadmap, hopefully for Q2: https://github.com/Budibase/budibase/projects/10

**Reply by CosmasSC128** [reply]
You mean there isn't? Thank you for the reply.

**Reply by melohagan** [reply]
Yes sorry, fixed the typo!

**Reply by CosmasSC128** [reply]
No worries, thank you for the reply.
